cli/src/command.rs

```diff
@@ -187,17 +187,14 @@ pub fn run() -> sc_cli::Result<()> {
 					let storage = backend.expose_storage();
 					unwrap_client!(client, cmd.run(config, client.clone(), db, storage))
 				}),
-				BenchmarkCmd::Overhead(cmd) => runner.sync_run(|config| {
-					use cere_client::benchmark_inherent_data;
-
-					let (client, _, _, _) = cere_service::new_chain_ops(&config)?;
-					let wrapped = client.clone();
-
-					let inherent_data = benchmark_inherent_data()
-						.map_err(|e| format!("generating inherent data: {:?}", e))?;
-
-					unwrap_client!(client, cmd.run(config, client.clone(), inherent_data, wrapped))
-				}),
+				BenchmarkCmd::Overhead(_cmd) => {
+					print!("BenchmarkCmd::Overhead is not supported");
```

Comment:
Please pay attention that the implementation of this handler is [postponed](https://www.notion.so/cere/Tech-debt-aab401cd2dc147ae91555b3f31e3fdae).

Reply:
Please, answer this:
- Did you investigate the purpose of these sub-commands? What are they being used for?
- How do we perform benchmarking on custom pallets at the moment?
- Is it safe to update the Mainnet without these Sub-commands?